src/iframe-entry/components/Confirmation.tsx

```diff
@@ -13,19 +13,24 @@ type IConfirmationProps = TFlexProps & {
     address: string;
     name: string;
     balance: string;
+    canConfirm?: boolean;
     onReject: MouseEventHandler<HTMLButtonElement>;
     onConfirm: MouseEventHandler<HTMLButtonElement>;
 };
 
 export const Confirmation: FC<IConfirmationProps> = ({
     address,
+    canConfirm,
     name,
     balance,
     children,
     onReject,
     onConfirm,
     ...rest
 }) => {
+    const isConfirmDisabled =
+        typeof canConfirm === 'undefined' ? false : !canConfirm;
```

Comment:
Why canConfirm may be undefined?

Reply:
<Confirmation> is also being used in contexts where there is no conditions to disable confirmation, so there is no need to pass this prop